### ISSUE TITLE
Dependency lookup for libadwaita-1 with method 'pkgconfig' failed

### ISSUE THREAD

**Issue by Natetronn** [mrvladus/Errands#255]
I get the following error when installing

> Errands/meson.build:7:0: ERROR: Dependency lookup for libadwaita-1 with method 'pkgconfig' failed: Invalid version, need 'libadwaita-1' ['>= 1.5'] found '1.4.4'.

Can the libadwaita version be lowered, perhaps?

**Comment by mrvladus** [comment]
It's using widgets that was added in libadwaita 1.5, so it just won't work with lower versions. What distro do you use? And why not just use flatpak that already have all dependencies?

**Comment by Natetronn** [comment]
Thanks for the reply. 

I'm on Arch, so I'm able to avoid Flatpack for the most part, thanks to the AUR (except for Bottles). Not a fan of installing ~730mb for every ~5mb package and then similar downloads for updates (with ~2.5Mbps downloads it's a bit slow.)


```
 1. [✓] io.github.mrvladus.List.Locale               stable           i            flathub             7.0 kB / 476.0 kB
 2. [✓] org.freedesktop.Platform.openh264            2.4.1            i            flathub           920.7 kB / 976.5 kB
 3. [✓] org.gnome.Platform.Locale                    46               i            flathub            18.1 kB / 367.5 MB
 4. [✓] org.gnome.Platform                           46               i            flathub           216.7 MB / 346.8 MB
 5. [✓] io.github.mrvladus.List                      stable           i            flathub             5.0 MB / 7.4 MB
```
 I realize it may make things easier on developers, though.

**Comment by mrvladus** [comment]
I thought arch already had libadwaita 1.5 in its repository

**Comment by Natetronn** [comment]
There is only one so far:
https://aur.archlinux.org/packages?O=0&K=libadwaita

I'm not able to install it without some hassle, though, since I'd have to remove the current version and all those packages that depend on it (or something: I tried with yay, but it failed.)

I see 1.5.0 is in testing on Manjaro, though, so maybe it will be out soon enough there:
https://packages.manjaro.org/?query=libadwaita

I'll either wait for that to be released or use the Flatpack, as you suggested. I've since noticed in the README that it's the only method you're currently supporting, so I appreciate the extra support having now realized that.

Thanks again!
